ios-repo-app/DeviceRepository.swift

```diff
@@ -0,0 +1,66 @@
+//
+//  DeviceRepository.swift
+//  ios-repo-app
+//
+//  Created by juanita aguilar on 10/11/19.
+//  Copyright © 2019 Cascadia College. All rights reserved.
+//
+
+import Foundation
+// TODO : Use Generics and typeAlias to make the Repository class more general
+class DeviceRepository:APIRepository {
```

Comment:
To make the repository code reusable, you can add a Generic dataype using
class Repository<T>:API Repository {

}
Then when you create a new repository, you can inform it of the type, like
var myRepo = Repository<Device>()